scaladoc/src/dotty/tools/scaladoc/tasty/comments/CommentExpander.scala

```diff
@@ -79,7 +79,7 @@ class CommentExpander {
       case None =>
         // SI-8210 - The warning would be false negative when this symbol is a setter
         if (ownComment.indexOf("@inheritdoc") != -1 && ! sym.isSetter)
-          println(s"${sym.span}: the comment for ${sym} contains @inheritdoc, but no parent comment is available to inherit from.")
+          report.warning(s"${sym.span}: the comment for ${sym} contains @inheritdoc, but no parent comment is available to inherit from.", sym.srcPos)
```

Comment:
Is is still useful to include `${sym.span}` in the message now that `report.warning` is used with `sym.srcPos`?  The message may be confusing to the user who may not understand what those numbers mean?

```
 -- Warning: scaladoc-testcases/src/tests/inheritDoc.scala:6:6 ------------------
6 |  def f: Unit = { }
  |      ^
  |<76..76>: the comment for method f contains @inheritdoc, but no parent comment is available to inherit from.
```

Reply:
You may be right. I didn't change the message, just the way it's sent, but probably we should remove span there.